cli/scan/scan.go

```diff
@@ -42,17 +46,20 @@ func scanMain(args []string, c cli.Config) (err error) {
 				return
 			}
 
-			fmt.Printf("Scanning %s...\n", host)
+			wg.Add(1)
+			go func(host string) {
+				fmt.Printf("Scanning %s...\n", host)
+				results, err := scan.Default.RunScans(host, c.IP, c.Family, c.Scanner, c.Timeout)
+				if err != nil {
+					log.Fatal(err)
```

Comment:
It's a bit annoying to have to `log.Fatal` here instead of just returning the error, but it accomplishes about the same result.